application-templates/starter/package.json

```diff
@@ -7,8 +7,8 @@
     "postinstall": "manypkg check",
     "build": "mc-scripts build",
     "start": "mc-scripts start",
-    "compile-html": "NODE_ENV=production mc-scripts compile-html",
-    "compile-html:local": "NODE_ENV=production MC_APP_ENV=development mc-scripts compile-html --transformer @commercetools-frontend/mc-dev-authentication/transformer-local.js",
+    "compile-html": "mc-scripts compile-html",
+    "compile-html:local": "MC_APP_ENV=development mc-scripts compile-html --transformer @commercetools-frontend/mc-dev-authentication/transformer-local.js",
```

Comment:
Making sure I follow along with the new `build` behaviour: why don't we `--build-only` here?

Reply:
You can always run `compile-html` after `build`, even if you didn't use `--build-only`. What happens is that the new `compile-html` overrides the previous version.